package.json

```diff
@@ -244,6 +245,18 @@
     "trailingComma": "es5",
     "printWidth": 100
   },
+  "remarkConfig": {
+    "settings": {
+      "commonmark": true,
+      "fences": true,
+      "listItemIndent": "1",
+      "rule": "-",
+      "ruleSpaces": false
+    },
+    "plugins": [
+      "remark-frontmatter"
```

Comment:
`remark-frontmatter` is here to allow `remark` to parse frontmatter blocks correctly